raml-docs/README.md

```diff
@@ -0,0 +1,8 @@
+# RAMLing
+
+* See: http://raml.org for more about the RAML
+* To build: 1) swallow your pride and get node.js; 2) `npm install -g raml2html`
+
+Including this here/now because I'm using it as a guide to develop a new
+API endpoint, we could abandon this or move it to a different repo.
+Let's chat!
```

Comment:
Okay, this is holy-crap-awesome and I already want to capture the keys endpoint docs this way, but I think oc_erchef is the right project for it since that's where the actual API lives.